pkg/utils/utils.go

```diff
@@ -409,6 +492,24 @@ func NeedUpdate(iface *sriovnetworkv1.Interface, ifaceStatus *sriovnetworkv1.Int
 }
 
 func configSriovDevice(iface *sriovnetworkv1.Interface, ifaceStatus *sriovnetworkv1.InterfaceExt) error {
+	var result error
+	if err := configPf(iface, ifaceStatus); err != nil {
+		log.Log.Error(err, "ConfigSriovInterfacesInParallel(): fail to configure sriov interface. resetting interface.", "address", iface.PciAddress)
+		result = err
+		if iface.ExternallyManaged {
+			log.Log.Info("ConfigSriovInterfacesInParallel(): skipping device reset as the nic is marked as externally created")
+		} else {
+			if resetErr := resetSriovDevice(*ifaceStatus); resetErr != nil {
+				log.Log.Error(resetErr, "ConfigSriovInterfacesInParallel(): failed to reset on error SR-IOV interface")
+				result = resetErr
+			}
+		}
+	}
+
+	return result
+}
+
+func configPf(iface *sriovnetworkv1.Interface, ifaceStatus *sriovnetworkv1.InterfaceExt) error {
```

Comment:
all the logs have `configSriovDevice()` that OK ?

Reply:
+1 please change the log function name